stl/inc/variant

```diff
@@ -987,11 +1001,12 @@ template <class Enable, class _Ty, class... _Types>
 struct _Variant_init_helper {}; // failure case (has no member "type")
 
 template <class _Ty, class... _Types>
-struct _Variant_init_helper<void_t<decltype(_Variant_init_overload_set<_Types...>{}(_STD declval<_Ty>()))>, _Ty,
+struct _Variant_init_helper<
+    void_t<decltype(_Variant_init_overload_set<_Types...>{}(std::declval<_Ty>(), std::declval<_Ty>()))>, _Ty,

```

Comment:
restore ` _STD` macro instead of `std::`

Reply:
Oops, I overlooked them. Thanks